vue3-ts-quasar/src/composables/github/useGists.ts

```diff
@@ -0,0 +1,21 @@
+import { Ref } from 'vue';
+import { parseQuery } from '@/components/GistsPanel/parseQuery';
+import { useQuery, useResult } from '@vue/apollo-composable';
+import { USER_GISTS_QUERY } from './queries';
+import { GistItem } from '../github/types';
+
+interface UseGists {
+  getUserGists: () => {
+    data?: GistItem;
+    loading: Ref<boolean>;
+  };
+}
+
+export const useGists = (): UseGists => {
+  const getUserGists = (): any => {
```

Comment:
Prefer strongly typing whenever possible. I think we should avoid the use of `any` whenever possible.